cedar-policy/src/api.rs

```diff
@@ -694,13 +694,13 @@ pub enum SchemaError {
     #[error("{0} is not a record")]
     ContextOrShapeNotRecord(ContextOrShape),
     /// An Action Entity (transitively) has an attribute that is an empty set
-    #[error("Action attribute is an empty set")]
-    ActionEntityAttributeEmptySet,
+    #[error("Action `{0}` attribute is an empty set")]
```

Comment:
Let's use the error message we have for Core, which is better:
```suggestion
    #[error("action `{0}` has an attribute that is an empty set")]
```

Reply:
I was wondering if we need this struct here. Could we just use the one from the validator instead?

Reply:
I think I'll just make non-breaking-changes in this PR and leave those to another PR.